src/torchmetrics/utilities/checks.py

```diff
@@ -758,14 +758,8 @@ def is_overridden(method_name: str, instance: object, parent: object) -> bool:
 
 
 def _in_doctest() -> bool:
-    """Determine if script running in doctest context."""
```

Comment:
Yeah, was testing this locally and didn't work much, so env var looks better...

Reply:
But then we don't need to have it as function?

Reply:
you are right, function removed :)